### PR TITLE
Add Arabic keyboard layout for Western Baluchi (bgn)

### PR DESCRIPTION
**Pull request by Mjbmr**

### CODE REVIEW

**Comment by kartikm** [comment]
Can #382 be close?

**Comment by Mjbmr** [comment]
Yes.

File: rules/bgn/bgn-arab.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+			['X', 'ه'],
+			['C', 'ژ'],
+			['V', 'ڑ'],
+			['B', 'ۆ'],
+			['N', 'ٔ'],
+			['M', 'ء'],
+			['<', '>'],
+			['>', '<'],
+			['\\?', '؟']
+		],
```
Comment (santhoshtr) [inline rules/bgn/bgn-arab.js]: Please remove this trailing comma

**Comment by santhoshtr** [comment]
Thanks for the contribution. Would it be possible to add a few test cases for this input method. See the tests folder, it is mostly self explanatory. Also see documentation of this project. Tests will help us to make sure the input method works as expected even if we dont know the language. Thanks!

**Comment by Mjbmr** [comment]
I did what you asked, please let me know if there is anything wrong with it.

**Comment by Mjbmr** [comment]
Any update?

**Pull request merged**